recipe/meta.yaml

```diff
@@ -39,7 +41,7 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
-    - cuda-version {{ major_version }}
+    - cuda-version {{ major_version }}.{{ minor_version }}
```

Comment:
Have tightened down the `cuda-version` pin to get components from the CTK version that matches `cuda-python`'s own version (at least in terms of major minor). Let the patch version float upwards